guides/features/pipecat-flows.mdx

```diff
@@ -335,7 +335,7 @@ Execute after LLM inference completes. Useful for:
 
 - **Pre-actions**: Execute immediately, before any LLM processing begins
 - **LLM Inference**: Processes the node's messages and functions
-- **Post-actions**: Execute only after LLM processing and TTS completion
+- **Post-actions**: Execute after LLM processing and TTS completion
```

Comment:
Hmm...I'm realizing now that I think the function actions might _sometimes_ be triggered a bit before the bot actually finishes audibly speaking...

Reply:
Can't really repro now. Something to keep an eye out for though.